ddd/word.rb

```diff
@@ -0,0 +1,22 @@
+require 'set'
+
+class Word
+  attr_reader :word, :article
+
+  @@all = []
+
+  def initialize(article, word)
+    @article = article
+    @word = word
+    @rating = 0
+    @@all << self
```

Comment:
While this certainly works, it's not good practice to use class variables. Avoid them if you can. They can lead to memory leak problems, because they're never automatically deleted.
Instead, you can use an instance of your `Set` class to keep the array of all the words.

Reply:
hm, I think I'll do that in another branch? I'll have to check how to use `Set`

Reply:
super easy, with the set ;)